Lib/test/test_io.py

```diff
@@ -1723,6 +1723,21 @@ def test_truncate(self):
         with self.open(support.TESTFN, "rb", buffering=0) as f:
             self.assertEqual(f.read(), b"abc")
 
+    def test_truncate_after_write(self):
+        with self.open(support.TESTFN, "wb") as f:
```

Comment:
Thank you. Can you add a comment referring to the bug entry at the beginning at the test?